cocos/ui/UIListView.cpp

```diff
@@ -455,42 +455,33 @@ void ListView::setDirection(Direction dir)
     ScrollView::setDirection(dir);
 }
     
-void ListView::requestRefreshView()
+void ListView::refreshView()
 {
-    _refreshViewDirty = true;
+    forceDoLayout();
 }
 
-void ListView::refreshView()
+void ListView::requestDoLayout()
+{
+    _innerContainerDoLayoutDirty = true;
+}
+
+void ListView::doLayout()
```

Comment:
@neokim 
The original code is 

```
void ListView::doLayout()		
{		
    Layout::doLayout();		
   refreshViewIfNecessary();		
 }
```

It does call `Layout::doLayout`, I think it's wrong.

Thanks for clarification.